Tests/Test Helpers/Misc Test Helpers.ahk

```diff
@@ -0,0 +1,59 @@
+﻿#NoEnv  ; Recommended for performance and compatibility with future AutoHotkey releases.
+; #Warn  ; Enable warnings to assist with detecting common errors.
+SendMode Input  ; Recommended for new scripts due to its superior speed and reliability.
+SetWorkingDir %A_ScriptDir%  ; Ensures a consistent starting directory.
+
+TEST_HELPER_BUILD_MINIMIZE_SCITE4AUTOHOTKEY_SEQUENCE(theCheckExistsTryLimitIntervalList){
+	; AHK gets confused when you reuse variable names sometimes that are allegedly global, so we gotta make them unique here in the tests
```

Comment:
This comment is outdated now that we properly put everything into functions, remove it